jormungandr/src/explorer/graphql/mod.rs

```diff
@@ -1046,19 +1047,23 @@ impl Epoch {
             None => return Ok(None),
         };
 
-        let epoch_lower_bound = block_on(
+        let epoch_lower_bound = block_on(async {
             context
                 .db
                 .get_block(&epoch_data.first_block)
-                .map(|block| u32::from(block.expect("The block to be indexed").chain_length)),
-        );
+                .await
+                .map(|block| u32::from(block.chain_length))
+        })
+        .unwrap();
```

Comment:
It's useful to retain that panic message from `expect` here instead of uninformative `unwrap`.

Reply:
yep, actually I updated it but the push was rejected....